superagi/controllers/agent.py

```diff
@@ -438,14 +442,14 @@ def get_agent_configuration(agent_id: int,
         dict: Agent configuration including its details.
 
     Raises:
-        HTTPException (status_code=404): If the agent is not found.
+        HTTPException (status_code=404): If the agent is not found or deleted.
     """
 
     # Define the agent_config keys to fetch
     keys_to_fetch = AgentTemplate.main_keys()
     agent = db.session.query(Agent).filter(agent_id == Agent.id).first()
 
```

Comment:
Can you update in the query itself rather than doing after fetching from DB?

Reply:
sure we can do this